### DISCUSSION TITLE
why do i get `#nuxt-test` instead of `#__nuxt` on the root `div`?

### DISCUSSION THREAD

**Discussion by stefanprobst** [danielroe/nuxt-vitest#258] [Q&A] [answered]
when adding `nuxt-vitest` >= 0.8.6 to nuxt config options.modules i get a root `<div id="nuxt-test">` instead of `<div id="__nuxt">` (which breaks my app's styling)

is that intentional?

**Comment by danielroe** [comment, marked as answer]
Unfortunately it's due to a bug in happy-dom which we have to work around by using a different id.

It's possible we don't need to do this with the new jsdom support

**Comment by vasilistotskas** [comment]
Is there any temporary fix for now ? i tried to override the `rootId: '__nuxt',` and also `domEnvironment: 'jsdom'` and i still get nuxt-test as root id .
